command-launcher-plugin/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>2.34</version>
+        <relativePath />
+    </parent>
+    <artifactId>command-launcher</artifactId>
+    <version>1.0-SNAPSHOT</version>
```

Comment:
IIUC it should be hosted as a plugin, right? I see no reason in keeping it as a part of Jenkins repo

Reply:
From PR comment:

> physical split & new plugin creation would happen post-merge